src/util/renderers/renderer.builder.ts

```diff
@@ -0,0 +1,25 @@
+import { DesktopRenderer } from './desktop.renderer.js'
```

Comment:
In the current format:
* this class has no value IMO because you would easier just directly instanciate `DesktopRenderer`  or `VbisualEditorRenderer`
* This was the interface and I stay to it: `rendererBuilder.createRenderer(Mediawiki: mediawiki_instance, RendererMode: auto|mobile|desktop|specific: [RendererAPI: visualEditor, DesktopRest, MobileRest])`
* Remove `render()`, should not be there IMO, it's push the confusion between `RendererBuilder` and `Renderer`

Reply:
1. Ok, I instantiate them immediately in `src/util/saveArticles.ts`
2. I believe this interface should be build when we start with mobilehtml. There is a problem with isMainPage condition, and I still didn't come up with a decision on how to apply it.
3. I removed this builder since isMainPage condition is no longer necessary till we apply renderers for mobile.

Reply:
Upd:
The interface of `rendererBuilder` has been implemented.